plugins/kubefedctl 

```diff
@@ -0,0 +1 @@
+repository = https://github.com/kvokka/asdf-kubefedctl
```

Comment:
This URL needs to be the URL to the Git repository, so it needs to end in `.git`.

Reply:
But there are a few examples in the repo, with no `.git` suffix.
Do you like the idea, it I'll do a fix for it?

Reply:
I think the reason URLs without the `.git` suffix work is because Github is being smart and serving the repo when it detects requests from git. If you open another PR I'll definitely merge it. This is one of those chores that has fallen through the cracks for a couple months now.